packages/vue-dot/package.json

```
@@ -35,7 +35,9 @@
 		"dev": "vue-cli-service serve",
 		"build": "ts-node --project tsconfig.json --files scripts/build.ts",
 		"lint": "vue-cli-service lint",
-		"test": "vue-cli-service test:unit --runInBand"
+		"test": "vue-cli-service test:unit --runInBand",
+		"test:e2e":"yarn cypress run-ct",
+		"test:e2e:gui":"yarn cypress open-ct"
```

```suggestion
		"test:e2e": "cypress run-ct",
		"test:e2e:gui": "cypress open-ct"
```
Il n'y a pas de commande `cypress` dans le `package.json`, je pense que tu voulais lancer `cypress` directement